src/res.rc

```diff
@@ -48,6 +48,8 @@ BEGIN
     VK_F7,      IDM_MOVE,       VIRTKEY
     VK_F8,      IDM_COPY,       VIRTKEY
     VK_F9,      IDM_COPYTOCLIPBOARD, VIRTKEY
+    VK_F10,     IDM_SYMLINK,    VIRTKEY
+    VK_F10,     IDM_HARDLINK,    VIRTKEY, SHIFT
     VK_DELETE,  IDM_DELETE,     VIRTKEY

```

Comment:
F10 is used for the context menu key; what about F11?

Reply:
Good point!
Fixed with 9d031f2